azurelinuxagent/common/protocol/wire.py

```diff
@@ -1181,10 +1150,6 @@ def get_on_hold(self):
         return self.get_goal_state().extensions_goal_state.on_hold
 
     def upload_logs(self, content):
-        host_func = lambda: self._upload_logs_through_host(content)
```

Comment:
_call_hostplugin_with_container_check() implements retry logic when the HGAP returns 410 (ResourceGone). Those retries are needed only for the extensionsArtifact API. Upload log does not neeed this.